### PR TITLE
Release v0.14.6 updates

### PR DESCRIPTION
**Pull request by mattkasun**

### CODE REVIEW

**Review by abhishek9686 — APPROVED** [review]

**Review by afeiszli — APPROVED** [review]

File: netclient/functions/upgrades/v0.14.-6.go [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+)
+
+var upgrade0146 = UpgradeInfo{
+	RequiredVersions: []string{
+		"v0.14.0",
+		"v0.14.1",
+		"v0.14.2",
+		"v0.14.3",
+		"v0.14.4",
+		"v0.14.5",
```
Comment (0xdcarns) [inline netclient/functions/upgrades/v0.14.-6.go:15]: this should be the only required version

Comment (0xdcarns) [reply netclient/functions/upgrades/v0.14.-6.go:15]: also the file name has a "-" in it, is it necessary?

**Review by 0xdcarns — CHANGES_REQUESTED** [review]

File: netclient/functions/upgrades/v0.14.-6.go (hunk 1 above)
Comment (mattkasun) [reply netclient/functions/upgrades/v0.14.-6.go:15]: file name updated to match v0.-14-5.go

Comment (mattkasun) [reply netclient/functions/upgrades/v0.14.-6.go:15]: > 

if someone is updating from an earlier version,  all the intermediate updates are needed.

**Review by afeiszli — APPROVED** [review]

**Pull request merged**